pkg/hubble/metrics/flows-to-world/handler.go

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of Cilium
+
+package flows_to_world
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	"github.com/cilium/cilium/pkg/hubble/metrics/api"
+	labels2 "github.com/cilium/cilium/pkg/labels"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type flowsToWorldHandler struct {
+	flowsToWorld *prometheus.CounterVec
+	context      *api.ContextOptions
+	worldLabel   string
+	anyDrop      bool
+	port         bool
+}
+
+func (h *flowsToWorldHandler) Init(registry *prometheus.Registry, options api.Options) error {
+	c, err := api.ParseContextOptions(options)
+	if err != nil {
+		return err
+	}
+	h.context = c
+	for key := range options {
+		switch strings.ToLower(key) {
+		case "any-drop":
+			h.anyDrop = true
+		case "port":
+			h.port = true
+		}
+	}
+	labels := []string{"protocol", "verdict"}
+	if h.port {
+		labels = append(labels, "port")
+	}
+	labels = append(labels, h.context.GetLabelNames()...)
+
+	h.flowsToWorld = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: api.DefaultPrometheusNamespace,
+		Name:      "flows_to_world_total",
+		Help:      "Total number of flows to reserved:world",
```

Comment:
This help message doesn't match with the documentation description of the metric, which just adds to my confusion at this point. :grin: